configuration/configuring-language/en.md

```diff
@@ -3,20 +3,40 @@ title = "Configuring Language"
 +++
 # Configuring Language
 
-You can change your language by going to the Budgie Menu, opening Settings, then clicking Region and Language.
+It is easy to change the language that was selected during the installation of Solus.
 
-You will now be shown the following dialog.
+## Budgie
+
+You can change your language in Budgie by opening the Budgie Menu and then clicking on "Region and Language" in the System Settings category.
+
+You will now be shown the following window.  This window allows you to change your language, formatting for dates and times and the language of your keyboard.
 
 {{< altimg "region-and-languages.jpg" "help-center/configuration/configuring-language/" >}}
 
-In this dialog, click Language. The following dialog will appear.
+Start by clicking "Language". This brings up a selection of languages to chose from.
```

Comment:
Uhh yea, I'm just gonna say I was hallucinating or something. <sup>Guessing I had the wrong window open, derp.</sup>